app/actions/service_credential_binding_create.rb

```diff
@@ -55,6 +55,18 @@ def complete_binding_and_save(binding, binding_details, last_operation)
         event_repository.record_create(binding, @user_audit_info, @audit_hash, manifest_triggered: false)
       end
 
+      def save_incomplete_binding(binding, broker_operation)
+        binding.save_with_attributes_and_new_operation(
+          {},
+          {
+            type: 'create',
+            state: 'in progress',
+            broker_provided_operation: broker_operation
+          }
+        )
+        event_repository.record_start_create(binding, @user_audit_info, @audit_hash, manifest_triggered: false)
```

Comment:
Ideally a function called `save...` would only save things, and not also create audit events. In future we might rename this to something like `record_binding_start()`.

Reply:
Good point - I will update the method name in the next story